roles/parallels/tasks/install.yml

```diff
@@ -80,22 +80,35 @@
         group: staff
         mode: '0644'
 
-- name: Enumerate Parallels auto-deploy package files
-  find:
-    path: '{{ parallels_install_cache }}/pd-autodeploy/'
-    pattern: '*.pkg'
-    file_type: directory
-    recurse: yes
-  register: available_pkgs
+- name: Get Parallels version from `pd-autodeploy.zip`
+  shell:
```

Comment:
Would it be better to add this to the `parallels_facts` module rather than having this much shell code embedded in a task?

Reply:
FWIW, I've been struggling to make this more elegant, so I've just sent the patch as is. It was tested in tandem with #8.

Reply:
I think this is fine, since we want the autodeploy package version, not the Parallels version.

However, I would adjust the task name and registered variable name to make it clear this is the autodeploy package version, not the Parallels version. They can be the same, but aren't always. For example, the current autodeploy package is version 18.1.0 while the current Parallels version is 18.1.1.

Reply:
Yes, exactly. I renamed the var.

> Wow, that's a pretty bad bug in the installer.

@samdoran It's funny that this collection has the same type of bug in the removed task. This new task fixes the same problem (unsorted `find: *.pkg`) happening on a different layer as a side effect, additionally to making sure there's one DMG in the installer dir.

Reply:
That's so funny. I made the same naive assumption of just one file. Doh.